docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+load_balancer:
+    image: tutum/haproxy
+    links:
+        - web
+
+web:
+    image: savaslabs/squabble
```

Comment:
This is OK for now but eventually I'd prefer to use a specific version number so it's clearer what is running in production, locally, etc